packages/clay-link/src/ClayLink.js

```diff
@@ -10,7 +10,30 @@ import templates from './ClayLink.soy.js';
  * Implementation of the Metal Clay Link.
  * @extends ClayComponent
  */
-class ClayLink extends ClayComponent {}
+class ClayLink extends ClayComponent {
+	/**
+	 * @inheritDoc
+	 */
+	attached() {
+		this.addListener('click', this._handleClick, true);
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	disposed() {
+		this.removeListener('click', this._handleClick);
+	}
+
+	/**
+	 * Handles click.
+	 * @param {!Event} event
+	 * @protected
+	 */
+	_handleClick(event) {
+		event.preventDefault();
```

Comment:
Hi @julien,

I force push on this commit, the change we talked about in our call.

If @matuzalemsteles doesn't agree with these changes, I would like to know if he could look into this and provide us another alternative with a clear example, because I feel really lost with it.

Regards.

Reply:
Hey @georgel-pop-lr @julien, on second thought about this `preventDefault` here, I think we have to remove it, we will be removing the default behavior of redirecting to another page everywhere we are using ClayLink, this will break a lot of behavior and there is no way to reverse the call to `preventDefault` when it's already been called, we're going to have to revert that.

I'll send you a PR updating, don't worry about it.

Reply:
@matuzalemsteles yes sure, makes sense, sorry for that - we tested so many things that I think unwanted stuff got merged, for example the clay-link [demo](https://github.com/liferay/clay/blob/aea7542a04801e6ad20206ab5724e85af586ae8f/packages/clay-link/demos/index.html#L108-L129) still has the `handleItemClicked` function

Reply:
No problem, I also didn't notice it being added here even though I saw it 😅.